excellent/functions/builtin_test.go

```diff
@@ -190,6 +190,10 @@ func TestFunctions(t *testing.T) {
 		{"datetime_diff", mdy, []types.XValue{xs("03-10-2019 1:00am"), xs("03-11-2019 1:00am"), xs("h")}, xi(23)},
 		{"datetime_diff", mdy, []types.XValue{xs("03-10-2019 1:00am"), xs("03-11-2019 1:00am"), xs("D")}, xi(1)},
 
+		{"datetime_from_epoch", dmy, []types.XValue{xn("1497286619.000000000")}, xdt(time.Date(2017, 6, 12, 16, 56, 59, 0, time.UTC))},
+		{"datetime_from_epoch", dmy, []types.XValue{ERROR}, ERROR},
+		{"datetime_from_epoch", dmy, []types.XValue{}, ERROR},
```

Comment:
Did this for your Rowan.

Reply:
dayum.. how did I ever let that get out of lexical order!